docs/amazon.aws.aws_az_info_module.rst

```diff
@@ -55,7 +55,7 @@ Parameters
                 <td>
                 </td>
                 <td>
-                        <div>AWS access key. If not set then the value of the AWS_ACCESS_KEY_ID, AWS_ACCESS_KEY or EC2_ACCESS_KEY environment variable is used.</div>
+                        <div><code>AWS access key</code>. If not set then the value of the AWS_ACCESS_KEY_ID, AWS_ACCESS_KEY or EC2_ACCESS_KEY environment variable is used.</div>
```

Comment:
I don't see any markup in the aws.py fragment that would have generated these code blocks, were they manually added?

Reply:
Thank you for your valuable feedback @tremble.